com.unity.render-pipelines.universal/Runtime/UniversalRenderer.cs

```diff
@@ -868,6 +868,10 @@ void CreateCameraRenderTarget(ScriptableRenderContext context, ref RenderTexture
                     colorDescriptor.autoGenerateMips = false;
                     colorDescriptor.depthBufferBits = (useDepthRenderBuffer) ? k_DepthStencilBufferBits : 0;
                     cmd.GetTemporaryRT(m_ActiveCameraColorAttachment.id, colorDescriptor, FilterMode.Bilinear);
+#if ENABLE_VR && ENABLE_XR_MODULE
+                    cmd.SetRenderTarget(m_ActiveCameraColorAttachment.id, 0, CubemapFace.Unknown, -1);
```

Comment:
Is this logic XR specific?
VR/XR macro will be set to true even when XR is not enabled(they detect XR module presence, not runtime). 
cameraData.xr.enabled should be checked instead.

Reply:
It only breaks in XR (UWP + WinMR) because of the contents for that platform is not black. Technically it could be for any it just depends. It does not break standalone windows so thus the preproc.
I will add cameraData.xr.enabled

Reply:
You're right. It could be for any platform. Removed the preprocs